src/views/virtualmachines/details/utils/types.ts

```diff
@@ -2,5 +2,6 @@ import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
 
 export type NavPageComponentProps = {
   instanceTypeExpandedSpec?: V1VirtualMachine;
+  obj: V1VirtualMachine;
```

Comment:
Except for obj, is no other prop guaranteed? Obj is guaranteed?

Reply:
Other than obj we get name, ns, params, plural.

Reply:
But there is no type specified in typescript sdk

Reply:
added those params